src/main/java/com/telefonica/iot/cygnus/sinks/OrionKafkaSink.java

```diff
@@ -258,7 +279,9 @@ private void persistAggregation(KafkaAggregator aggregator) throws Exception {
         if (!topicAPI.topicExists(zookeeperClient, topicName)) {
             LOGGER.info("[" + this.getName() + "] Creating topic " + topicName
                     + " at OrionKafkaSink");
-            topicAPI.createTopic(zookeeperClient, topicName, new Properties());
+            LOGGER.info("[" + this.getName() + "] Partitions: " + partitions + ", "
```

Comment:
I would add the partitions number and the replication factor to the previous logging trace. Something like "Creating topic at OrionKafkaSink. Topic name: <name>, partitions: <partitiions>, replication factor: <replication_factor>".